docs/.vuepress/config.js

```diff
@@ -77,29 +78,51 @@ module.exports = {
             '/getting-started/prerequisites'
           ]
         },
-        'core-functionality',
-        'api',
+        {
+          title: 'Regen Ledger',
+          collapsable: false,
+          children: [
+            '/regen-ledger/',
+            '/regen-ledger/interfaces',
+          ]
+        },
         {
           title: 'Modules',
+          collapsable: false,
           children: [
             {
               title: 'Data Module',
               collapsable: false,
               children: [
                 '/modules/data/',
-                {title: 'Protobuf Documentation', path: '/modules/data/protobuf', collapsable: true}
+                {
+                  title: 'Protobuf Documentation',
+                  path: '/modules/data/protobuf'
+                }
               ]
             },
             {
               title: 'Ecocredit Module',
               collapsable: false,
               children: [
                 '/modules/ecocredit/',
-                {title: 'Protobuf Documentation', path: '/modules/ecocredit/protobuf', collapsable: true}
+                {
+                  title: 'Protobuf Documentation',
+                  path: '/modules/ecocredit/protobuf'
+                }
               ]
             },
           ]
-        }
+        },
+        {
+          title: 'Tutorials',
+          collapsable: false,
+          children: [
+            '/tutorials/',
+            // '/tutorials/data-cli',
+            // '/tutorials/data-grpc',
```

Comment:
Does it make sense to keep these files for later revisions or would it be better to remove them entirely?

Reply:
sure.  I don't see an issue with keeping the files as long as we don't link to them.